File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        k = 0
        j = 0

        while j < len(nums):

            first = j
            while j < len(nums) and nums[first]==nums[j]:
                j+=1
            nums[k] = nums[first]
            k+=1
            
        return k 
        

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] output = new int[nums.length*2];
        int n = nums.length;
        for (int i =0;i<n;i++){
            output[i] = nums[i];
            if (i + n < 2*n){
                output[i+n] = nums[i];
            }
        }
        return output;
    }
}